### DISCUSSION TITLE
Blank screen after compiling my app

### DISCUSSION THREAD

**Discussion by nappalm** [cawa-93/vite-electron-builder#898] [Q&A] [answered]
My application was working fine until I started the build process with the default command built into the project.
It is worth mentioning that I made an adjustment to generate my _.exe_ file and it was to remove `--dir` from the compilation command
`    "compile": "cross-env MODE=production npm run build && electron-builder build --config .electron-builder.config.js --config.asar=false",
`

After installing through .exe my application stayed blank. I have noticed that the calls to api are made successfully, could it be because I am using [chakra-ui](https://chakra-ui.com/)? 🤨

![image](https://user-images.githubusercontent.com/35016608/215672684-3285fa31-4945-41be-be55-d60be79a1b20.png)

**Comment by cawa-93** [comment, marked as answer]
App compiled without `--dir` works perfectly for me. In your case, issue, probably, in something else

**Reply by nappalm** [reply]
It turned out to be a problem with `react-router-dom` 🤨

**Reply by Yusef-Adel** [reply]
@nappalm how did you fix it i'm facing same issue

**Reply by nappalm** [reply]
@Yusef-Adel, if you're utilizing react-router-dom, consider employing `<HashRouter>`
